### ISSUE TITLE
USWDS - Feature: Enhancing Screen Reader Accessibility - Announcing the Current Page in Breadcrumb Navigation

### ISSUE THREAD

**Issue by rachelhsmith** [uswds/uswds#5100]
### Is your feature request related to a problem? Please describe.

An accessibility audit of our site has identified an opportunity to enhance the usability for screen reader users. Currently, the final breadcrumb element is not announced as the current page by most screen readers as it is not a focusable element, so aria-current=“page” is ignored.

![breadcrumbs-uswds](https://user-images.githubusercontent.com/117450473/212877990-a42417fa-aad6-4aba-9663-4ebdf4362b17.png)
Current breadcrumbs with static element as last item.

### Describe the solution you'd like

A solution would be to add an `<a>` tag inside the final `<li>` element of the breadcrumb and add aria-current=“page” to this. Making the last item a link would make sure that the active page is announced to the user so there is no doubt to the user which page they are on.

![breadcrumbs-on-gov-uk](https://user-images.githubusercontent.com/117450473/212878341-d033457d-343f-4699-99cc-0bbf00d8fe06.png)
Example of focusable link element as final breadcrumb item.

### Additional context

An example of this in MDN docs https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Feature+Request%22) for duplicate feature requests.

**Comment by mejiaj** [comment]
Confirming. Current page isn't being read on VoiceOver. Tested in [StorybookJS Preview →](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/develop/iframe.html?args=&id=components-breadcrumb--default&viewMode=story)

**Comment by iamjolly** [comment]
Hi, and thanks for your patience here. I know you submitted this back in early 2023 and @mejiaj confirmed it as a bug in 2024. Glad to see the ongoing attention to it.

I started investigating this and did some further testing. After looking at it closely, I think this USWDS breadcrumb component is actually working like it should. I think what you’re reporting here might come from a misunderstanding of how screen readers interact with breadcrumb navigation. When tabbing through a component, users might miss the current page item if it’s not an interactive link. But reading in order through the list items correctly announces the `aria-current="page"` state.

I made a video [(attached)](https://github.com/user-attachments/assets/e26a1fe4-76ff-4196-9a7f-b8b202fb2a15) showing how this works in VoiceOver using Safari. You can hear the `aria-current="page"` for the breadcrumb list items announced as expected in it, both on the component page and in the example you brought up in this issue.

When it comes to the solution you suggest — using an empty anchor element to create a focusable target — that’s not an approach we’d recommend. Introducing empty or non-functional anchor tags can create separate usability and accessibility challenges — that can add unnecessary tab stops that don't provide functional value, or possibly not add a working tab stop. If the `href` attribute is missing from the anchor element, that can create confusion since anchors without an `href` attribute won’t show up in the natural tab order ([this article from Dennis Deacon about missing href links](https://www.dennisdeacon.com/accessibility/when-is-a-link-not-a-link/#:~:text=An%20anchor%20element%20without%20an,meaningful%20behavior%20to%20assistive%20technologies.) explains this further).

We truly appreciate everyone helping out to improve accessibility here. After doing this testing, we’ve confirmed that we think the current version is the best choice for accessibility, but if you have additional thoughts or findings, we’d definitely be interested in hearing more.

Thanks again for your contribution!
